### PR TITLE
WIP: Easy build bat

### PR DESCRIPTION
**Pull request by GokuWorks**
Create bat file to easily build release.

### CODE REVIEW

**Comment by GokuWorks** [comment]
I think I fixed the UI env but feel free to let me know if its still wrong.

File: Easy Build Release.bat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+git status
```
Comment (RobbeBryssinck) [inline Easy Build Release.bat:1]: Not needed

File: Easy Build Release.bat [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+git fetch
```
Comment (RobbeBryssinck) [inline Easy Build Release.bat:2]: This does not do anything on its own except fetch metadata. These git operations can be done by the programmer.

File: Easy Build Release.bat [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+git submodule update --recursive
```
Comment (RobbeBryssinck) [inline Easy Build Release.bat:3]: Likewise, submodules should not always be updated automatically, this is a conscious choice. Updating submodules should not be in the build script.

**Review by RobbeBryssinck — CHANGES_REQUESTED** [review]
Filename should be "Build.bat".

**Pull request merged**